components/nodemanager-service/pgdb/node-managers-process-node.go

```diff
@@ -127,20 +127,12 @@ func (db *DB) ProcessIncomingNode(node *manager.NodeMetadata) error {
 		return errors.Wrap(err, "ProcessIncomingNode unable to marshal projects data")
 	}
 
-	var mgrType string
-	if node.ManagerId != "" {
```

Comment:
now that the type is included in the report, we dont have to look up the type in the db